tests/test_interfaces.py

```diff
@@ -5,22 +5,27 @@
 from pathlib import Path
 from itertools import product
 
+import pytest
+
 import cv2
 
 from nwb_conversion_tools import interface_list
 from nwb_conversion_tools import NWBConverter
 from nwb_conversion_tools import MovieInterface
 
 
-def test_interface_schemas():
-    for data_interface in interface_list:
-        # check validity of source schema
-        schema = data_interface.get_source_schema()
-        Draft7Validator.check_schema(schema)
+@pytest.mark.parametrize("data_interface", interface_list)
+def test_interface_source_schema(data_interface):
+    # check validity of source schema
+    schema = data_interface.get_source_schema()
+    Draft7Validator.check_schema(schema)
+
```

Comment:
@bendichter do we gain any speed by outsourcing the loop to this fixture? Otherwise it just feels less explicit to me

Reply:
Even if there isn't a difference in speed, this is easier to read imo

Reply:
this separates out each data interface into its own test, which helps you identify which ones are failing.